lib/compat/wordpress-6.0/client-assets.php

```diff
@@ -61,7 +61,11 @@ function gutenberg_resolve_assets() {
 
 	foreach ( $block_registry->get_all_registered() as $block_type ) {
 		if ( ! empty( $block_type->style ) ) {
-			$style_handles[] = $block_type->style;
+			if ( is_array( $block_type->style ) ) {
+				$style_handles[] = $block_type->style[0];
```

Comment:
Why only take the first one?

Reply:
For reference: [Answer](https://github.com/WordPress/wordpress-develop/pull/2853#discussion_r906078875)